index.html

```diff
@@ -18,14 +18,17 @@
       <!-- Start: Navbar-->
 
       <header class="container">
-        <nav>
+        <nav id="navbar" class="nav-mbl-inactive">
           <a class="logo" href="#">My Logo</a>
+          <i class="fa-solid fa-x close-menu"></i>
           <ul>
-            <li><a href="#portfolio">Portfolio</a></li>
-            <li><a href="#about-section">About</a></li>
-            <li><a href="#contact-section">Contact</a></li>
+            <li><a class="nav-links menu1" href="#portfolio">Portfolio</a></li>
```

Comment:
- [ ] Great job on designing the mobile menu! However, I kindly request that you ensure when users attempt to click on the portfolio, it redirects them to the works section :+1:.